### PR TITLE
badge test rename

### PR DESCRIPTION
**Pull request by jspaaks**
**List of related issues or pull requests**

Refs: #295

**Describe the changes made in this pull request**

Updated the repository names and name of paths used for live testing the cli, after
- https://github.com/fair-software/badge-test changed its name to https://github.com/fair-software/howfairis-livetest and 
- https://githlab.com/jspaaks/badge-test changed its name to https://gitlab.com/jspaaks/howfairis-livetest

Also set the sleep duration to 0 for gitlab platfaorm, API rate limit there is plenty high

**Instructions to review the pull request**

```shell
git clone <this repo>
git checkout <this branch>
cd howfairis
python3 -m venv venv3
source venv3/bin/activate
pip3 install --editable --no-cache-dir .[dev]
bash clitests/script.sh
```

### CODE REVIEW

File: clitests/script.sh [hunk 1]
```diff
@@ -60,22 +59,24 @@ echo -e "\n\n"
...
 
 # github
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p force/00100"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p force/10110"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p force/11110"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p force/11111"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p force/uu1uu"
-eval_and_print_status "howfairis https://github.com/fair-software/badge-test -p ignore_commented_badges"
+SLEEP_DURATION=20
```
Comment (fdiblen) [inline clitests/script.sh:62]:
```suggestion
SLEEP_DURATION=0
```

Comment (jspaaks) [reply clitests/script.sh:62]: Agree, result will not be changed by sleeping anyway

**Comment by jspaaks** [comment]
Github action status for this branch [livetests](https://github.com/fair-software/howfairis/actions/workflows/livetests.yml)

**Review by fdiblen — APPROVED** [review]
LVGTM :name_badge:

**Comment by jspaaks** [comment]
Thanks!

**Pull request merged**